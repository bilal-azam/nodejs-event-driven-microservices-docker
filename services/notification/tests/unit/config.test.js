const config = require('../../src/environment/config');

describe('config', () => {
  test('should load default values', () => {
    expect(config.name).not.toBeNull();
    expect(config.environment).not.toBeNull();
    expect(config.messagebus).not.toBeNull();
    expect(config.email.service).not.toBeNull();
    expect(config.email.username).not.toBeNull();
    expect(config.email.password).not.toBeNull();
    expect(config.email.adminEmailID).not.toBeNull();
    expect(config).toMatchSnapshot();
  });
});
