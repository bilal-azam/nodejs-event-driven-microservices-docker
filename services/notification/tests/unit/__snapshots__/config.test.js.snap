// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config should load default values 1`] = `
Object {
  "email": Object {
    "adminEmailID": "",
    "password": "",
    "service": "",
    "username": "",
  },
  "environment": "dev",
  "messagebus": "amqp://guest:guest@localhost",
  "name": "Notification Service",
  "startedMessage": "Notification Service is running",
}
`;
